pkgs/tools/games/slipstream/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, fetchFromGitHub, stdenv, makeWrapper, buildMaven, maven, jdk8 }:
+let
+  settings = (buildMaven ./project-info.json).settings;
```

Comment:
Maven packages in nixpkgs are built using `maven.buildMavenPackage` (aka double invocation) these days. You might want to check [this package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/keycloak/scim-keycloak-user-storage-spi/default.nix) as an example.

Reply:
Had to rebase from master but done :) it's not possible to do the `finalAttrs` pattern with `buildMavenPackage`, is it? only `stdenv.mkDerivation` I assume. If not, is there a better pattern than wrapping the whole derivation in `rec`?